### DISCUSSION TITLE
tal 2.00 bug?

### DISCUSSION THREAD

**Discussion by AA111A1AZA1AZXC1** [community/community#69830] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

friends
hello

I installed tal.2.00. It happens that it starts to analyze and after a few minutes it reads the analyzes and the following message appears. THIS PROGRAM HAS ABORTED

Maybe there is some way to fix this problem. hugs
dirceu

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @AA111A1AZA1AZXC1! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I've gone ahead and moved this to the correct category for you.

When you say you installed tal.2.00, are you referring to https://tal-software.com/products/tal-reverb? It may help users in this community who may be familiar with this software if you're explicit about the software that you're using and if you share more information about how you're running into this error. 

Good luck!

**Comment by AA111A1AZA1AZXC1** [comment]
dear sarto I installed 2.00 eas on the website itself, where the installation of 2 engines appears. I went to GitTube and I couldn't find the location you put me in. OK hugs dirceu
________________________________
De: Sarto Jama ***@***.***>
Enviado: quarta-feira, 11 de outubro de 2023 18:10
Para: community/community ***@***.***>
Cc: AA111A1AZA1AZXC1 ***@***.***>; Mention ***@***.***>
Assunto: Re: [community/community] tal 2.00 bug? (Discussion #69830)


Hey there! 👋

Thanks for posting in the GitHub Community, @AA111A1AZA1AZXC1<https://github.com/AA111A1AZA1AZXC1>! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I've gone ahead and moved this to the correct category for you.

When you say you installed tal.2.00, are you referring to https://tal-software.com/products/tal-reverb? It may help users in this community who may be familiar with this software if you're explicit about the software that you're using and if you share more information about how you're running into this error.

Good luck!
